catalog/dags/providers/provider_api_scripts/smk.py

```diff
@@ -135,7 +137,7 @@ def get_record_data(self, data: dict) -> dict | None:
             "foreign_landing_url": foreign_landing_url,
             "license_info": license_info,
             "title": self._get_title(data),
-            "url": url,
+            "url": url.replace(" ", "%20"),
```

Comment:
Would it be useful to keep the record of which providers have the spaces in the URLs encoded?

Reply:
Hm, I'm not sure what we would use that information for, but if we end up needing it it should be pretty easy to see which providers had this logic added. Particularly since we've been handling it as errors arise.